src/control/server/drpc.go

```diff
@@ -0,0 +1,140 @@
+//
+// (C) Copyright 2019 Intel Corporation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// GOVERNMENT LICENSE RIGHTS-OPEN SOURCE SOFTWARE
+// The Government's rights to use, modify, reproduce, release, perform, display,
+// or disclose this software are subject to the terms of the Apache License as
+// provided in Contract No. 8F-30005.
+// Any reproduction of computer software, computer software documentation, or
+// portions thereof marked with this legend must also reproduce the markings.
+//
+
+package main
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/daos-stack/daos/src/control/drpc"
+	"github.com/golang/protobuf/proto"
+	"github.com/pkg/errors"
+)
+
+var sockFileName = "daos_server.sock"
+
+func getDrpcClientSocket(sockDir string) string {
+	return filepath.Join(sockDir, "daos_io_server.sock")
```

Comment:
We will need to figure out how handle once we have multiple io servers setup. Since both of them can't listen on daos_io_server.sock

Reply:
I think we have a Jira ticket to deal with this already, actually.